### DISCUSSION TITLE
Should "achromatomaly" be included as a color filter in the a11y add-on?

### DISCUSSION THREAD

**Discussion by blakegearin** [storybookjs/storybook#18752] [General]
One of the color filters currently provided in the a11y add-on is [achromatomaly](https://github.com/storybookjs/storybook/blob/85f7cddea03239313f9a03f651e6080b7dfe4e85/addons/a11y/src/components/ColorFilters.tsx#L55-L61). A user by the name of [Chromaphobe ](https://chromaphobe.com/)recently released their findings ([video](https://www.youtube.com/watch?v=kYZ00B5O_VQ), [transcript & footnotes](https://chromaphobe.com/how-a-made-up-color-blindness-infiltrated-the-internet/)) about the origin of this term and why it's not [blue cone monochromacy](https://en.wikipedia.org/wiki/Blue_cone_monochromacy), as some sources have suggested.

Additionally, I noticed the percentage provided by Storybook for achromatomaly ([`0,00001%`](https://github.com/storybookjs/storybook/blob/2dcc9dabe2f4f6736fe5fa6ede2f829fc93126d5/addons/a11y/src/components/VisionSimulator.tsx#L22)) seems to have come from [this PR comment](https://github.com/storybookjs/storybook/issues/15866#issuecomment-900516640) which links to Reddit, a questionable source. None of the other sources listed appear to mention achromatomaly.

For reference, the earliest [commit](https://github.com/storybookjs/storybook/commit/7c4d54486d7fab9c06fa5495d175af2d2d5abe6c) and [PR](https://github.com/storybookjs/storybook/pull/4086) I could find for achromatomaly came from @ndelangen in 2018. Presumably the SVG filters were brought in from a different source, though it's unclear where. Regardless, Storybook's filter does line up with [Colorjack's](http://web.archive.org/web/20081014161121/http://www.colorjack.com/labs/colormatrix/), which has since been retracted.

Storybook's value: `0.618, 0.320, 0.062, 0, 0 0.163, 0.775, 0.062, 0, 0 0.163, 0.320, 0.516, 0, 0 0, 0, 0, 1, 0`
Colorjack's value: `R:[61.8, 32, 6.2], G:[16.3, 77.5, 6.2], B:[16.3, 32.0, 51.6]`

With all this in mind, I'm of the opinion that "achromatomaly" should be removed from Storybook or at the least evaluated further for validity.

Related issues:

- #5959
- #15866

**Comment by ndelangen** [comment]
Hello @blakegearin I'm absolutely no expert on this, and have also not tracked what my source was for the values.

I'm ok with removing / updating the values, either is fine by me!
Would you be able to open a PR?

**Reply by blakegearin** [reply]
Hi! I'm no expert either, just been researching this domain recently. That's why I opened a discussion instead of going straight to PR. Since you mentioned it I went ahead and created one though: #18852

Perhaps others who have worked on this addon have some thoughts. Tagging for visibility: 
@tooppaaa @maya @metasean @jsomsanith @Aarbel @MichaelArestad

**Reply by MichaelArestad** [reply]
Thanks for the tag. I am also okay with a PR. My preference would be to remove it for now.

**Reply by yannbf** [reply]
I'd love to hear your thoughts on this @mlama007
